### PR TITLE
 Added interval argument to make_progress_bar()

### PR DESCRIPTION
**Pull request by Rarst**
Allows to set update interval, available in class constructor but not previously accepted by function wrapper.

### CODE REVIEW

File: php/utils.php [hunk 1]
```diff
@@ -511,14 +511,15 @@ function mustache_render( $template_name, $data = array() ) {
  *
  * @param string  $message  Text to display before the progress bar.
  * @param integer $count    Total number of ticks to be performed.
+ * @param int     $interval  The interval in milliseconds between updates.
```
Comment (gitlost) [inline php/utils.php]: Extra space before the "The". Also could you expand the comment to
```
Optional. The interval in milliseconds between updates. Default 100.
```
?

Comment (Rarst) [reply php/utils.php]: I copy/pasted the doc from the implementing class, would you like wording updated there as well for consistency?

Comment (gitlost) [reply php/utils.php]: Well, it'd be nice, but probably not worth it!

Comment (Rarst) [reply php/utils.php]: Yeah, the class is upstream in PHP CLI tools so not worth it. :)

**Pull request merged**